canonical-web-and-design/juju.is
# Add missing social links to footer

## Done

- Added social icons for chat, forum and careers
- Added footer social links for forum and careers

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- In the footer, click each of the social links, make sure they work
